css/app.scss

```diff
@@ -10,6 +10,9 @@
 
  */
 
+@import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+@import '../node_modules/bootstrap/dist/css/bootstrap-theme.min.css';
```

Comment:
be careful. When the URL explicitly ends with `.css`, Sass will generate a CSS `@import` statement instead of doing a Sass import. You probably want to omit the extension here, which would perform an actual Sass import.

Reply:
Removing the extension introduces path errors with bootstrap's font files, but cannot find an `@import` in generated css file though.